bson/raw_bson.py

```diff
@@ -13,6 +13,46 @@
 # limitations under the License.
 
 """Tools for representing raw BSON documents.
+
+Inserting and Retrieving RawBSONDocuments
+=========================================
+
+Example: Moving a document between different databases/collections
+.. code-block:: python
```

Comment:
Does this render correctly? I feel like you might need to have a newline before `.. code-block::` to get it to render correctly?

You can build the docs locally by installing `sphinx` and running `python setup.py doc`

Reply:
Ah you're right, thanks for noticing!